### PR TITLE
WRQ-4709 : Added performance tests for agate components

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added performance tests for agate components

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-4709

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

File: src/views/agate/DateTimePicker.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import kind from '@enact/core/kind';
```
Comment (stanca-pop-lgp) [inline src/views/agate/DateTimePicker.js:1]: Please sort imports.

File: src/views/agate/Drawer.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import Button from "@enact/agate/Button";
```
Comment (stanca-pop-lgp) [inline src/views/agate/Drawer.js:1]: Please use single quotes for imports.

File: src/views/agate/Dropdown.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import kind from '@enact/core/kind';
```
Comment (stanca-pop-lgp) [inline src/views/agate/Dropdown.js:1]: Please sort imports.

File: src/views/agate/Icon.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import kind from '@enact/core/kind';
```
Comment (stanca-pop-lgp) [inline src/views/agate/Icon.js:1]: Please use agate/Icon and sort imports.

File: src/views/agate/LabeledIcon.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import LabeledIcon from '@enact/agate/LabeledIcon';
+import kind from '@enact/core/kind';
+
+const LabeledIconView = kind({
+	name: 'LabeledIconView',
+
+	render: () => (
+		<LabeledIcon
```
Comment (stanca-pop-lgp) [inline src/views/agate/LabeledIcon.js:8]: Please sort props.

File: src/views/agate/LabeledIconButton.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import LabeledIconButton from '@enact/agate/LabeledIconButton';
+import kind from '@enact/core/kind';
+
+const LabeledIconButtonView = kind({
+	name: 'LabeledIconButtonView',
+
+	render: () => (
+		<LabeledIconButton
```
Comment (stanca-pop-lgp) [inline src/views/agate/LabeledIconButton.js:8]: Please sort props.

File: src/views/agate/MediaPlayer.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import kind from '@enact/core/kind';
```
Comment (stanca-pop-lgp) [inline src/views/agate/MediaPlayer.js:1]: Please sort imports.

File: src/views/agate/Panels.js [hunk 8]
```diff
@@ -0,0 +1,70 @@
+import Button from '@enact/agate/Button';
+import Header from '@enact/agate/Header';
+import Item from '@enact/agate/Item';
+import {Panel, Panels} from '@enact/agate/Panels';
+import TabGroup from "@enact/agate/TabGroup";
```
Comment (stanca-pop-lgp) [inline src/views/agate/Panels.js:5]: Please use single quotes for imports.

File: src/views/agate/Picker.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+	'Banana',
+	'Clementine',
+	'Durian'
+];
+
+const PickerView = kind({
+	name: 'PickerView',
+
+	render: () => (
+		<Picker id="picker" title="Picker Default" >{pickerList}</Picker>
```
Comment (stanca-pop-lgp) [inline src/views/agate/Picker.js:15]: Please remove empty space after title prop value.

File: src/views/agate/PopupMenu.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+	const handleToggle = useCallback(() => {
+		setOpen(!open);
+	}, [open]);
+
+	return (
+		<>
+			<Button id="button-open" onClick={handleToggle}>open</Button>
+			<PopupMenu
+				id="popupMenu"
+				open={open}
```
Comment (stanca-pop-lgp) [inline src/views/agate/PopupMenu.js:17]: Please sort props.

File: src/views/agate/RangePicker.js [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+
+const RangePickerView = kind({
+	name: 'RangePickerView',
+
+	render: () => (
+		<RangePicker
+			id="rangePicker"
+			max={100}
+			min={0}
+			defaultValue={0}
```
Comment (stanca-pop-lgp) [inline src/views/agate/RangePicker.js:12]: Please sort props.

File: src/views/agate/Scroller.js [hunk 12]
```diff
@@ -0,0 +1,18 @@
+import Scroller from '@enact/agate/Scroller';
+import kind from '@enact/core/kind';
+
+const ScrollerView = kind({
+	name: 'ScrollerView',
+
+	render: () => (
+		<div style={{height: '700px'}} id="scroller">
```
Comment (stanca-pop-lgp) [inline src/views/agate/Scroller.js:8]: Please sort props.

File: src/views/agate/Slider.js [hunk 13]
```diff
@@ -0,0 +1,15 @@
+import Slider from '@enact/agate/Slider';
+import kind from '@enact/core/kind';
+
+const SliderView = kind({
+	name: 'SliderView',
+
+	render: () => (
+		<>
+			Slider
+			<Slider id="slider" min={0} max={100} defaultValue={0} />
```
Comment (stanca-pop-lgp) [inline src/views/agate/Slider.js:10]: Please sort props.

File: src/views/agate/TimePicker.js [hunk 14]
```diff
@@ -0,0 +1,12 @@
+import TimePicker from '@enact/agate/TimePicker';
+import kind from '@enact/core/kind';
+
+const TimePickerView = kind({
+	name: 'TimePickerView',
+
+	render: () => (
+		<TimePicker id="timePicker"	/>
```
Comment (stanca-pop-lgp) [inline src/views/agate/TimePicker.js:8]: Please leave a single empty space before closing tag.

File: src/views/agate/ToggleButton.js [hunk 15]
```diff
@@ -0,0 +1,16 @@
+import ToggleButton from '@enact/agate/ToggleButton';
+import kind from '@enact/core/kind';
+
+const ToggleButtonView = kind({
+	name: 'ToggleButtonView',
+
+	render: () => (
+		<ToggleButton
+			id="agate-togglebutton"
+			toggleOnLabel="On"
```
Comment (stanca-pop-lgp) [inline src/views/agate/ToggleButton.js:10]: Please sort props.

File: src/views/agate/VirtualList.js [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+			dataSizeProp = parseInt(urlDataSize);
+		}
+
+		populateItemsArray(dataSizeProp);
+
+		return (
+			<div style={{height: '700px'}}>
+				<VirtualList
+					id="VirtualList"
+					dataSize={dataSizeProp}
```
Comment (stanca-pop-lgp) [inline src/views/agate/VirtualList.js:54]: Please sort props.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]
Great job! Effort appreciated.

**Review by stanca-pop-lgp — APPROVED** [review]

**Review by seunghoh — APPROVED** [review]

**Pull request merged**